src/boot/store.js

```diff
@@ -2,20 +2,21 @@
 import { AsyncStorage } from 'react-native';
 import { applyMiddleware, compose, createStore } from 'redux';
 import { persistStore, autoRehydrate } from 'redux-persist';
+import Reactotron from 'reactotron-react-native';
 
 import config from '../config';
 import rootReducer from './reducers';
 import middleware from './middleware';
 
 // AsyncStorage.clear(); // use to reset storage during development
 
-// uncomment the following lines to integrate reactotron with redux
-// const store = Reactotron.createStore(
-//   rootReducer,
-//   compose(autoRehydrate(), applyMiddleware(...middleware)),
-// );
+let store; 
 
-const store = compose(applyMiddleware(...middleware), autoRehydrate())(createStore)(rootReducer);
+if(__DEV__) {
+  store = Reactotron.createStore(rootReducer, {},compose(applyMiddleware(...middleware), autoRehydrate()))
```

Comment:
Since Reactotron creates a lightweight wrapper around our store, its no harm switching to normal store creation in production environment althought it is not really necessary because reactotron.createStrore() does not add much of an overhead.